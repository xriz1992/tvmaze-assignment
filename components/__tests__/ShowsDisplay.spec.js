import { shallowMount } from '@vue/test-utils'
import ShowsDisplay from '../ShowsDisplay'

const factory = () => {
  return shallowMount(Logo, {})
}

describe('Logo', () => {
  test('mounts properly', () => {
    const wrapper = factory()
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('renders properly', () => {
    const wrapper = factory()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
