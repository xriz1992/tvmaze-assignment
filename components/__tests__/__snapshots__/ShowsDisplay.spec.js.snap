// Jest Snapshot v1, https://goo.gl/fbAQLP

<div class="allShows"><h3 class="allShows--title">Action</h3><div class="shows"><button class="allShows--scrollLeft"><span>&lt;</span></button><!--[--><article class="allShows__showCard"><a href="/shows/2" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/163/407679.jpg" alt="Person of Interest" class="allShows__showCard--image" width="232" height="376"></div><h3>Person of Interest</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 11.999999999999993% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/4" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/213/534017.jpg" alt="Arrow" class="allShows__showCard--image" width="232" height="376"></div><h3>Arrow</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 25.999999999999996% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/6" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/257/642675.jpg" alt="The 100" class="allShows__showCard--image" width="232" height="376"></div><h3>The 100</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 22% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/11" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/189/474715.jpg" alt="Gotham" class="allShows__showCard--image" width="232" height="376"></div><h3>Gotham</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 22% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/13" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/444/1112175.jpg" alt="The Flash" class="allShows__showCard--image" width="232" height="376"></div><h3>The Flash</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 23% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/15" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/0/154.jpg" alt="Constantine" class="allShows__showCard--image" width="232" height="376"></div><h3>Constantine</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 24.000000000000004% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/18" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/423/1058582.jpg" alt="The Amazing Race" class="allShows__showCard--image" width="232" height="376"></div><h3>The Amazing Race</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 30% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/19" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/445/1114097.jpg" alt="Supernatural" class="allShows__showCard--image" width="232" height="376"></div><h3>Supernatural</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 15.999999999999996% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/21" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/164/412464.jpg" alt="The Last Ship" class="allShows__showCard--image" width="232" height="376"></div><h3>The Last Ship</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 24.000000000000004% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/24" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/213/532575.jpg" alt="Hawaii Five-0" class="allShows__showCard--image" width="232" height="376"></div><h3>Hawaii Five-0</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 24.000000000000004% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/26" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/178/446780.jpg" alt="Hellsing Ultimate" class="allShows__showCard--image" width="232" height="376"></div><h3>Hellsing Ultimate</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 22% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/29" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/286/715906.jpg" alt="Vikings" class="allShows__showCard--image" width="232" height="376"></div><h3>Vikings</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 14.000000000000004% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/31" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/142/356776.jpg" alt="Marvel's Agents of S.H.I.E.L.D." class="allShows__showCard--image" width="232" height="376"></div><h3>Marvel's Agents of S.H.I.E.L.D.</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 20% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/38" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/170/426759.jpg" alt="Z Nation" class="allShows__showCard--image" width="232" height="376"></div><h3>Z Nation</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 28% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/44" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/128/322484.jpg" alt="Scorpion" class="allShows__showCard--image" width="232" height="376"></div><h3>Scorpion</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 27% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/49" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/402/1007484.jpg" alt="Brooklyn Nine-Nine" class="allShows__showCard--image" width="232" height="376"></div><h3>Brooklyn Nine-Nine</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 19.000000000000004% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/59" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/426/1065450.jpg" alt="Chicago Fire" class="allShows__showCard--image" width="232" height="376"></div><h3>Chicago Fire</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 19.000000000000004% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/60" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/421/1053458.jpg" alt="NCIS" class="allShows__showCard--image" width="232" height="376"></div><h3>NCIS</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 20.999999999999996% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/72" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/444/1110924.jpg" alt="NCIS: Los Angeles" class="allShows__showCard--image" width="232" height="376"></div><h3>NCIS: Los Angeles</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 22% 0 0);"></div></span></article><article class="allShows__showCard"><a href="/shows/73" class=""><div><img src="https://static.tvmaze.com/uploads/images/medium_portrait/424/1061900.jpg" alt="The Walking Dead" class="allShows__showCard--image" width="232" height="376"></div><h3>The Walking Dead</h3></a><span class="allShows__showCard--rate"><div class="relative z-30 w-[100px]"><img src="/_nuxt/components/stars.webp" class="absolute inset-0" aria-hidden="true"><img src="/_nuxt/components/stars-filled.webp" class="absolute inset-0" width="100" height="100" aria-hidden="true" style="clip-path:inset(0 20% 0 0);"></div></span></article><!--]--><button class="allShows--scrollRight"><span>&gt;</span></button></div></div>